52.md

```diff
@@ -90,9 +90,12 @@ The list of tags are as follows:
 * `end` (optional) exclusive end Unix timestamp in seconds. If omitted, the calendar event ends instantaneously.
 * `start_tzid` (optional) time zone of the start timestamp, as defined by the IANA Time Zone Database. e.g., `America/Costa_Rica`
 * `end_tzid` (optional) time zone of the end timestamp, as defined by the IANA Time Zone Database. e.g., `America/Costa_Rica`. If omitted and `start_tzid` is provided, the time zone of the end timestamp is the same as the start timestamp.
+* `about` (optional) description of the calendar event
+* `image` (optional) url of an image to use for the event
```

Comment:
I've added support for `image` in Comingle.

Reply:
I'm changing the `about` field to `summary` in the next release of Corny Chat